client/templates/layout/footer/footer.html

```diff
@@ -32,12 +32,11 @@
 
       <div class="col-xs-5 col-sm-5 col-md-5 col-lg-5">
       <ul class="list-unstyled">
-          <li><a href="" class="fa fa-twitter-square fa-2x pull-right" data-event-category="link" data-event-action="link-click" data-event-label="external link click" data-event-value="twitter"></a></li>
```

Comment:
It seems like I might need to put extra options in reactio-social package for these analytics events class.